### PR TITLE
Add workshop

### PR DESCRIPTION
**Pull request by catarinamachado**

### CODE REVIEW

File: _i18n/pt.yml [hunk 1]
```diff
@@ -349,6 +349,14 @@ speakers:
       onde começou por trabalhar em tratamento dos dados com tecnologias de
       Business Intelligence. Atualmente é developer na área de AI, estando
       neste momento a trabalhar em Forecasting de Vendas.
+  joao_martins:
+    about: |
+      With a licentiate degree in Computer Science by Faculdade de
```
Comment (nelsonmestevao) [inline _i18n/pt.yml]: Here, it should be in Portuguese

File: _i18n/pt.yml [hunk 2]
```diff
@@ -620,13 +628,22 @@ activities:
...
   apresentacao_bosch:
     name: Apresentação da Bosch
-  ##TODO
   rally:
     name: Rally das Tascas
     about: O tradicional Rally das Tascas, evento social que pretende promover o convívio entre os participantes da SEI.
   workshop_smithmicro:
-    name: Workshop - SmithMicro
-    about: TBA
+    name: THE ILLUSION OF LIFE - A LESSON IN ANIMATION
```
Comment (nelsonmestevao) [inline _i18n/pt.yml]: I think it shouldn't be all caps.

File: _i18n/pt.yml (hunk 1 above)
Comment (catarinamachado) [reply _i18n/pt.yml]: noooo 😭😭

Comment (catarinamachado) [reply _i18n/pt.yml]: i will do it

File: _i18n/pt.yml [hunk 3]
```diff
@@ -620,13 +628,22 @@ activities:
...
-  ##TODO
   rally:
     name: Rally das Tascas
     about: O tradicional Rally das Tascas, evento social que pretende promover o convívio entre os participantes da SEI.
   workshop_smithmicro:
-    name: Workshop - SmithMicro
-    about: TBA
+    name: The Illusion of Life - A Lesson in Animation
+    about: |-
+      In this workshop we'll be working out computer graphics as a tool
```
Comment (nelsonmestevao) [inline _i18n/pt.yml]: Do you wanna do the same here? 🙃

**Review by nelsonmestevao — APPROVED** [review]

File: _i18n/pt.yml (hunk 3 above)
Comment (catarinamachado) [reply _i18n/pt.yml]: 😂done!

Comment (catarinamachado) [reply _i18n/pt.yml]: @nelsonmestevao still approved?

Comment (nelsonmestevao) [reply _i18n/pt.yml]: :rocket:

**Pull request merged**